### PR TITLE
use of ".map()" should be ".forEach()"

### PR DESCRIPTION
**Pull request by dcsmith926**
In "seed/challenges/json-apis-and-ajax.json", the use of ".map()" in the 5th and 6th challenges doesn't make sense -- we're not creating a new array by modifying each element of the response, and in challenge 4, the learner was told to use ".forEach()". Seeing ".map()" instead for no apparent reason would likely be a source of confusion.

### CODE REVIEW

**Comment by ltegman** [comment]
Thanks! These must have gotten overlooked when the challenge before was recently changed.
